PlatformRoot.js

```diff
@@ -82,10 +82,29 @@ for (let i = 0; i < process.argv.length; i++) {
 run(initialWorkspace)
 
 async function run(initialWorkspace) {
-    PL.app = require('./Platform/PlatformApp.js').newPlatformApp()
-    await PL.app.run(initialWorkspace)
-    console.log('Superalgos Platform App is Running!')
+    let PlatformApp = require('./Platform/PlatformApp.js');
```

Comment:
Rather than hard coding dependencies into PlatformApp the dependencies are passed in which allows for easy test setup and mocking.